src/Chronicle/ChronicleConfig.cs

```diff
@@ -0,0 +1,6 @@
+namespace Chronicle
+{
+    public class ChronicleConfig : IChronicleConfig
```

Comment:
Should this be public or internal since it's a default implementation?

Reply:
It can be internal, thanks for catching it.

And no problem at all, just wanted to get your attention when I saw you approved a couple other PRs. I hope everything turned out well! Happy holidays!